### ISSUE TITLE
[BUG] npm deprecate throws "422 Unprocessable Entity" error

### ISSUE THREAD

**Issue by sunknudsen** [npm/cli#2581]
### Current Behavior:

The following error is thrown.

```shell
npm ERR! code E422
npm ERR! 422 Unprocessable Entity - PUT https://registry.npmjs.org/npm-sync - Unprocessable Entity
```

### Expected Behavior:

`npm-sync` versions bellow `2.0.0-beta` are deprecated.

### Steps To Reproduce:

```console
$ npm deprecate npm-sync@"<2.0.0-beta" "Package repurposed" 
This operation requires a one-time password.
Enter OTP: ******
npm ERR! code E422
npm ERR! 422 Unprocessable Entity - PUT https://registry.npmjs.org/npm-sync - Unprocessable Entity
```

### Environment:

- OS: macOS Catalina 10.15.7
- Node: 14.15.1
- npm: 6.14.11

**Comment by jantimon** [comment]
I have the same issue - all I could find is: https://npm.community/t/npm-deprecate-giving-error-e422-unprocessable-entity/2337

**Comment by philippkuehn** [comment]
I have the same issue for `npm deprecate @tiptap/vue` (which is on version `2.0.0-alpha.13`)

**Comment by jcayzac** [comment]
Same here:

```sh
$ npm deprecate stringformat "This package has been rendered useless by string literals, and may present unknown vulnerabilities. Please stop depending on it."
npm ERR! code E422
npm ERR! 422 Unprocessable Entity - PUT https://registry.npmjs.org/stringformat - Unprocessable Entity
```

### log

```txt
28 verbose stack HttpErrorGeneral: 422 Unprocessable Entity - PUT https://registry.npmjs.org/stringformat - Unprocessable Entity
28 verbose stack     at /Users/julien.cayzac/.volta/tools/image/npm/7.11.2/node_modules/npm-registry-fetch/check-response.js:134:15
29 verbose statusCode 422
30 verbose pkgid stringformat@latest
```

### environment

```sh
$ npm --version
7.11.2
$ sw_vers
ProductName:	macOS
ProductVersion:	11.2.3
BuildVersion:	20D91
```

**Comment by darcyclarke** [comment]
npm `v6` is no longer in active development; We will continue to push security releases to `v6` at our team's discretion as-per our [Support Policy](https://github.com/npm/cli/wiki/Support-Policy).

If your bug is preproducible on `v7`, please re-file this issue using our [new issue template](https://github.com/npm/cli/issues/new?assignees=&labels=Bug%2CNeeds+Triage%2CRelease+7.x&template=bug_7.yml&title=%5BBUG%5D+%3Ctitle%3E).

If your issue was a feature request, please consider opening a new [RRFC](https://github.com/npm/rfcs/issues) or [RFC](https://github.com/npm/rfcs). If your issue was a question or other idea that was not CLI-specific, consider opening a discussion on our [feedback repo](https://github.com/npm/feedback)

**Comment by ohager** [comment]
Commenting: v7 works fine, while v6 for mentioned reason not

**Comment by zloirock** [comment]
The same now on 8.6.0.

![image](https://user-images.githubusercontent.com/2213682/166663586-37893fc4-fb46-45af-b939-f542874ea130.png)

**Comment by gustavo-m-vieira** [comment]
@zloirock Did you were able to bypass this? Its happening to me now.

**Comment by zloirock** [comment]
@gustavo-m-vieira yes, IIRC with updated version of NPM.

**Comment by inomdzhon** [comment]
Hi, @darcyclarke 

What should happens if I will try to deprecate already deprecated package version?

In my case, I'm testing GitHub Actions for deprecate npm package. For testing it, I try to deprecate https://www.npmjs.com/package/@vkontakte/icons/v/2.11.2 that actually deprecated, because I don't have package for deprecate now 😅 

There is my workflow file https://github.com/VKCOM/.github/blob/c5b9fd2950943b00dc570f37875837341ff89952/.github/workflows/npm_deprecate_package.yml#L27-L39

There is error
```
  npm deprecate "@vkontakte/icons@2.11.2" "Actually it is 'v1.11.2'. We made a typo in the past and released it as 'v2.11.2'. Sorry. It happens to everyone :)"
  shell: /usr/bin/bash -e {0}
  env:
    NPM_CONFIG_USERCONFIG: /home/runner/work/_temp/.npmrc
    NODE_AUTH_TOKEN: ***
npm ERR! code E[4](https://github.com/VKCOM/icons/actions/runs/4003881501/jobs/6872465124#step:3:4)22
npm ERR! 422 Unprocessable Entity - PUT https://registry.npmjs.org/@vkontakte%2ficons - Unprocessable Entity
```

### Environment

- node: v18.13.0
- npm: 8.19.3

**Comment by pcfreak30** [comment]
This just happened to me in v11.7.0 with a scoped package.
